### DISCUSSION TITLE
Checkbox field

### DISCUSSION THREAD

**Discussion by mattcox** [community/community#12661] [Projects and Issues]
We currently can add fields of type "Text", "Number", "Single Select", "Date" and "Iteration" to a project view. However, I think we're missing a field type - which is a standard boolean property that displays as a checkbox.

In our project, we want a field that marks if an issue has been merged into the main release branch. This would be a property that would be set by a workflow when the linked pull request is merged.

The checkbox field would ideally have the ability to set a default value - true or false.

**Comment by 700software** [comment]
We would like a checkbox that default to false which indicates whether we've manually sorted/dragged it to the correct order yet - since new issues default to the very end. Using single select now which is two clicks each time instead of one. :(
